tutorials/old/cross_correlation_from_sac.ipynb

```diff
@@ -6,6 +6,10 @@
    "source": [
     "# Calculate cross correlation functions from raw noise data in SAC\n",
     "\n",
+    "This notebook demontrates the original functionalities of Noisepy from Jiang and Denolle (2019). The core functions are identical to today's version of Noisepy. The difference is that this notebook does not use DataStores for data management and reads directly from ASDF and SAC.\n",
```

Comment:
Does this notebook still work?

Reply:
i have not tested it, so i moved it to "old"